build.gradle

```diff
@@ -393,5 +391,6 @@ tasks.verifyGoogleJavaFormat {
   exclude 'build/**'
   exclude 'vgen/build/**'
   exclude 'vgen/.apt_generated/**'
+  exclude 'src-gen/**'
```

Comment:
@garrettjonesgoogle I'm not 100% what creates this directory, but it appears on my workstation often (it might be running `artman` that does it). Without adding this line, `g test` fails.